pages/bookmarks/index.js

```diff
@@ -0,0 +1,67 @@
+import Tip from "../../components/Tip";
+import {ListItem} from "../../components/TipsList";
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function Bookmarks({tips, toggleBookmarking}) {
+  return (
+    <>
+      <StyledHeader>
+        <StyledTitle>Your Bookmarked Tips</StyledTitle>
+      </StyledHeader>
+      <StyledBody>
+        <h3>Here you can see the tips that you have bookmarked:</h3>
+        {!tips.find(tip => tip.isBookmarked) ? (
```

Comment:
are you sure you need the ! in fron of tips here?

Reply:
Hi @MerleMak ! 

Thanks a bunch for checking this PR and sharing your feedback. 

I've just implemented changes for all pointed items, except for the last one (if I remove the "!" in front of the `tips.find` method, the text `You have not bookmarked any tip yet.` is not being displayed when no cards have been bookmarked).

Cheers!